modules/popup/react/lib/usePopupStack.ts

```diff
@@ -66,12 +66,12 @@ export const usePopupStack = <E extends HTMLElement>(
         ? target.current || undefined
         : target
       : undefined;
-    PopupStack.add({element: ref.current!, owner: targetEl});
-    const element = ref.current!;
+    PopupStack.add({element: localRef.current!, owner: targetEl});
+    const element = localRef.current!;
     return () => {
       PopupStack.remove(element);
     };
-  }, [ref, target, popupRef]);
+  }, [localRef, target, popupRef]);
```

Comment:
I don't think refs are technically needed anymore by the lint rule if they are locally defined, but it doesn't hurt other than unnecessary reference comparisons.